src/util/languageUtils.js

```diff
@@ -0,0 +1,47 @@
+import url from 'url';
+import Accepts from 'accepts';
+
+export const LANG_DEFAULT = 'en-US';
+
+export const getLanguage = (request, supportedLangs, defaultLang=LANG_DEFAULT) => {
+	const firstPathComponent = request.url.path.split('/')[1];
+	const urlLang = supportedLangs.includes(firstPathComponent) ? firstPathComponent : null;
+	const browserLang = Accepts(request).language(supportedLangs);
+
+	const langSourcePreference = [
```

Comment:
So if i'm reading this structure right, it checks the url being request first for a language request, then the browser for a language cookie set and if nothing uses the default lang... This is probably premature question, but what about language preferences from the member's profile?